marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubConfigImpl.java

```diff
@@ -914,16 +899,7 @@ public DatabaseClient newStagingManageClient(String databaseName) {
     }
 
     public DatabaseClient newFinalManageClient() {
-        AppConfig appConfig = getAppConfig();
-        DatabaseClientConfig config = new DatabaseClientConfig(appConfig.getHost(), finalPort, appConfig.getRestAdminUsername(), appConfig.getRestAdminPassword());
-        config.setDatabase(finalDbName);
-        config.setSecurityContextType(SecurityContextType.valueOf(finalAuthMethod.toUpperCase()));
-        config.setSslHostnameVerifier(finalSslHostnameVerifier);
-        config.setSslContext(finalSslContext);
-        config.setCertFile(finalCertFile);
-        config.setCertPassword(finalCertPassword);
-        config.setExternalName(finalExternalName);
-        return appConfig.getConfiguredDatabaseClientFactory().newDatabaseClient(config);
+        return newStagingClient(finalDbName);
```

Comment:
Can the methods public DatabaseClient newStagingManageClient(String databaseName)  and public DatabaseClient newFinalManageClient() be removed as we don't use them ?

Reply:
Yes absolutely.